docs/guides/lint-arazzo.md

```diff
@@ -0,0 +1,73 @@
+# Lint Arazzo with Redocly CLI
+
+Arazzo is an open standard for describing multi-step API calls in sequence.
+While under development, this standard was also known as "OpenAPI Workflows".
+Redocly CLI offers support for checking that your Arazzo files are valid.
+
+{% admonition type="info" name="Experimental Arazzo support" %}
+This feature is at an early stage, please send us lots of feedback!
```

Comment:
Where should they send feedback? Recommend mentioning that here.

Reply:
Linked to our issues create page